org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -378,8 +387,17 @@ public static Location toLocation(IClassFile classFile, int offset, int length)
 		} catch (URISyntaxException e) {
 			JavaLanguageServerPlugin.logException("Error generating URI for class ", e);
 		}
-		Range range = toRange(classFile, offset, length);
-		return new Location(uriString, range);
+		return uriString;
+	}
+
+	public static String toUri(ITypeRoot typeRoot) {
+		if (typeRoot instanceof ICompilationUnit) {
+			return getFileURI((ICompilationUnit) typeRoot);

```

Comment:
should we rename getFileURI to toUri, to be consistent with other methods? If not that then rename toUri(IClassFile) to getFileURI